src/lib/api/api.go

```diff
@@ -467,6 +468,50 @@ func (a *API) ModifyIntervalDuration(w http.ResponseWriter, r *http.Request) {
 	a.send(w, a.marshalled())
 }
 
+// Edit or remove a route from the config screen.
+func (a *API) ModifyConfigRoutes(w http.ResponseWriter, r *http.Request) {
+	if err := r.ParseForm(); err != nil {
+		panic(err)
+	}
+	var req struct {
+		Action      string `json:"action"`
+		NewRoute    string `json:"newRoute"`
+		ActualRoute string `json:"actualRoute"`
+	}
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&req); err != nil {
+		panic(err)
+	}
+	switch req.Action {
+	case "edit-route":
+		for i, route := range a.config.Config.Routes {
+			if route.URL == req.ActualRoute {
+				a.config.Config.Routes[i] = config.Route{
+					Method: a.config.Config.Routes[i].Method,
+					URL:    req.NewRoute,
+					Header: a.config.Config.Routes[i].Header,
+					Body:   a.config.Config.Routes[i].Body,
+					Params: a.config.Config.Routes[i].Params,
+				}
+			}
+		}
+	case "delete-route":
+		for i, route := range a.config.Config.Routes {
+			if route.URL == req.ActualRoute {
+				a.config.Config.Routes = append(a.config.Config.Routes[:i], a.config.Config.Routes[i+1:]...)
+			}
+		}
+	}
+	_, err := a.config.Write()
+	if err != nil {
+		a.ResponseStatus = http.StatusText(400)
+		panic(err)
+	}
```

Comment:
```

	if _, err := a.config.Write(); err != nil {
		a.ResponseStatus = http.StatusText(400)
		panic(err)
	}
```
also, if you panic, there won't be a `ResponseStatus` sent to the UI. So, rather than panicking, go with logging an error so that it sends the UI.